internal/common/gamestate/gamestate.go

```diff
@@ -30,8 +23,8 @@ type GameState struct {
 	Environment    disasters.Environment
 	DeerPopulation foraging.DeerPopulationModel
 
-	// IIGOInfo contains states of the base versions of all roles
-	IIGOInfo IIGOBaseRoles
+	// IIGO History
+	IIGOHistory *[]shared.Accountability
```

Comment:
IIRC I commented on the pointer state of this before it was resolved? was it resolved? 

I don't see why this should be a pointer